src/Compute/Compute/Extension/RemoveAzureVMExtensionCommand.cs

```diff
@@ -58,6 +58,11 @@ public class RemoveAzureVMExtensionCommand : VirtualMachineExtensionBaseCmdlet
         [ValidateNotNullOrEmpty]
         public SwitchParameter Force { get; set; }
 
+        [Parameter(
+            Mandatory = false,
+            HelpMessage = "Run cmdlet in the background")]
+        public SwitchParameter AsJob { get; set; }
```

Comment:
based on other cmdlets that use this AsJob, do you need do anything else with this parameter in this code?

Reply:
No in my understanding nothing else is needed